modules/exploits/windows/local/ntusermndragover.rb

```diff
@@ -0,0 +1,96 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/file'
+require 'msf/core/post/windows/priv'
+require 'msf/core/post/windows/registry'
+require 'msf/core/exploit/exe'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = NormalRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Windows::ReflectiveDLLInjection
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Microsoft Windows NtUserMNDragOver Local Privilege Elevation',
+      'Description'    => %q(
+        This module exploits a null pointer deference in win32k triggered by the
+        NTUserMNDragOver syscall.
+        After receiving the menu window object returned by the window procedure function,
+        the xxxMNFindWindowFromPoint function does not effectively check the validity of
+        its member tagPOPUPMENU, causing the subsequent MNGetpItemFromIndex function to
+        trigger the NULL pointer deference.
+        This module has been tested against Windows 7 x86.
+      ),
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Clément Lecigne', # discovery
+          'Grant Willcox',   # exploit
+          'timwr'            # msf module
+        ],
+      'Platform'       => 'win',
+      'SessionTypes'   => ['meterpreter'],
+      'Targets'        =>
+        [
+          ['Windows 7 x86', { 'Arch' => ARCH_X86 }]
+        ],
+      'References'     =>
+        [
+          ['CVE', '2019-0808'],
+          ['URL', 'https://github.com/exodusintel/CVE-2019-0808'],
+          ['URL', 'https://github.com/ze0r/cve-2019-0808-poc'],
+          ['URL', 'http://blogs.360.cn/post/RootCause_CVE-2019-0808_EN.html'],
+          ['URL', 'https://blog.exodusintel.com/2019/05/17/windows-within-windows/'],
+        ],
+      'DisclosureDate' => 'Nov 26 2018',
```

Comment:
Uhhh shouldn't this be 03/12/2019, the date the advisory came out from MS? Not sure how this should be 2018 considering the bug came out in 2019.